src/features/footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import {ListItem } from '@mui/material';
```

Comment:
i would say you have made this component complex just by having style in a separate file, like even if you want to have styles in a separate file use classes or style, in this way I have to style the file to see what doing on in the component, like Link, List, these should be in the components